app/views/partials/view-claim/closed-claim-actions/manage-overpayments.html

```diff
@@ -0,0 +1,62 @@
+{% if displayHelper.getClaimStatusClosed(Claim['Status'], Claim['IsAdvanceClaim'], Claim['DateOfJourney']) %}
+
+    <form action="/claim/{{ Claim['ClaimId'] }}/update-overpayment-status" method="post" class="form">
+      {% include "partials/csrf-hidden-input.html" %}
+      <input type="hidden" name="lastUpdated" value="{{ Claim['lastUpdatedHidden'] }}">
+      
+      {% if Claim['IsOverpaid'] != true %}
+        {% set overPaymentTitleText = 'Amount' %}
+        {% set overPaymentInput = 'overpayment-amount' %}
+      {% elseif Claim['IsOverpaid'] == true %}
```

Comment:
Things like this can probably just be an `else` ?

Reply:
Or flip so you do the `true` check first, then everything else will set it otherwise.